sinks/api/requests.go

```diff
@@ -154,3 +154,30 @@ func (req deleteSinkReq) validate() error {
 
 	return nil
 }
+
+type validateReq struct {
+	Name         string         `json:"name,omitempty"`
+	Backend      string         `json:"backend,omitempty"`
+	Config       types.Metadata `json:"config,omitempty"`
+	Description  string         `json:"description,omitempty"`
+	Tags         types.Tags     `json:"tags,omitempty"`
+	ValidateOnly bool           `json:"validate_only"`
```

Comment:
We should remove this now because we have the separate validate endpoint instead. Also remove from updateReq and addReq